vaapi/vaapicontext.cpp

```diff
@@ -84,14 +86,21 @@ YamiStatus VaapiConfig::create(const DisplayPtr& display,
     VAConfigID config;
 
     if (!checkProfileCompatible(display, profile)){
-        ERROR("Unsupport profile");
+        std::stringstream msg;
+        msg << "Unsupported profile: " << profile << std::endl;
+        ERROR("%s", msg.str().c_str());
```

Comment:
`ERROR("Unsupported profile: %s", toString(profile).c_str());` would be simpler.